flake.nix

```diff
@@ -67,7 +92,10 @@
             buildEnv {
               paths = [
                 ros-core
-                turtlesim
+                rmw-fastrtps-cpp
+                rmw-cyclonedds-cpp
```

Comment:
wait, do you really need both ?

Reply:
Arf that is a copy of the docker we install for AGIMUS. @Kotochleb must know better than me if this is true.

Reply:
So fast rtps is the default one and AFAIK the only one that supports shared memmory, which makes default configuration faster. Cyclone on the other hand has better defaults for network connection and handles better cases where for example you have two network interfaces. Additionally CycloneDDS works much better with multi computer setups.

Generally I prefer to stay with FastDDS as a default for as long as possible, and change to CycloneDDS when FastDDS stops working.

Reply:
Ok, so you confirm it is not a mistake to want both, depending on use cases ?

Reply:
Seems so I will add it back.